content/docs/dvclive/get-started.md

```diff
@@ -157,7 +152,7 @@ Running stage 'train':
 ### Compare and Visualize DVC Experiments
```

Comment:
Another non-blocking follow up: should we make these tabs? It's a bit long and feels like a lot of space to just show different interfaces for the experiments table.